src/test/scala/org/broadinstitute/dsde/firecloud/integrationtest/OntologySearchSpec.scala

```diff
@@ -0,0 +1,106 @@
+package org.broadinstitute.dsde.firecloud.integrationtest
+
+import com.typesafe.scalalogging.slf4j.LazyLogging
+import org.broadinstitute.dsde.firecloud.integrationtest.ESIntegrationSupport._
+import org.scalatest.{BeforeAndAfterAll, FreeSpec, Matchers}
+
+class OntologySearchSpec extends FreeSpec with Matchers with BeforeAndAfterAll with LazyLogging with SearchResultValidation {
+
+  override def beforeAll = {
+    // use re-create here, since instantiating the DAO will create it in the first place
+    searchDAO.recreateIndex()
+    // make sure we specify refresh=true here; otherwise, the documents may not be available in the index by the
+    // time the tests start, leading to test failures.
+    logger.info("indexing fixtures ...")
+    searchDAO.bulkIndex(OntologySearchTestFixtures.fixtureDocs, refresh = true)
+    logger.info("... fixtures indexed.")
+  }
+
+  override def afterAll = {
+    searchDAO.deleteIndex()
+  }
+
+  /*
+    OntologySearchTextFixtures has five datasets, with these ontology nodes:
+
+    CSA_9220  : central sleep apnea < sleep apnea < sleep disorder < disease of mental health < disease
+    E_4325    : Ebola hemorrhagic fever < viral infectious disease < disease by infectious agent < disease
+    L_1240    : leukemia < hematologic cancer < immune system cancer < organ system cancer < cancer < disease of cellular proliferation < disease
+    HC_2531   : hematologic cancer < immune system cancer < organ system cancer < cancer < disease of cellular proliferation < disease
+    D_4       : Disease
+    None      : no doid
+   */
```

Comment:
I would suggest one more case to test which is for a node that has multiple parents at the same distance (order) away. "http://purl.obolibrary.org/obo/DOID_0050696" is a good one for that.